functions.php

```diff
@@ -1,4 +1,53 @@
 <?php
 
+require_once 'dbConnect.php';
 
+/**
+ * function to call a var dump with pre tags
+ * @param $variabletoDump - the variable to be dumped
+ */
+function var_dumpPre($variabletoDump) {
+    echo '<pre>';
```

Comment:
is this function just for testing? The only other place i can see it is commented out on index.php